explainer.md

```diff
@@ -587,15 +483,50 @@ Finally, to focus an accessible node, call its focus() method.
 virtualNode.focus();
 ```
 
-When a virtual accessible node is focused, input focus in the DOM is
-unchanged. The focused accessible node is reported to assistive technology
-and other accessibility API clients, but no DOM events are fired and
-document.activeElement is unchanged.
+When a virtual accessible node is focused, 
+input focus in the DOM is unchanged. 
+The focused accessible node is reported to assistive technology
+and other accessibility API clients, 
+but no DOM events are fired and document.activeElement is unchanged.
```

Comment:
hm, i wonder if that will come back to bite us... Just curious if folks would want virtualfocus/virtualblur events.

Reply:
Yeah maybe...